kerastuner/engine/hyperparameters.py

```diff
@@ -1054,11 +1054,18 @@ def value_to_cumulative_prob(value, hp):
             return 0.75
         return 0.25
     elif isinstance(hp, Choice):
+        if not hp.values:
+            return 0
         ele_prob = 1 / len(hp.values)
         index = hp.values.index(value)
         # Center the value in its probability bucket.
         return (index + 0.5) * ele_prob
     elif isinstance(hp, (Int, Float)):
+        if value < hp.min_value or value > hp.max_value:
+            raise ValueError('`value` must be in the range defined in the '
+                             'HyperParameter.')
```

Comment:
I think this check is already done at the sampling function.
We should assume the value is in the range.